index.js

```diff
@@ -48,6 +48,10 @@ var pluralTypes = {
     if (n % 10 === 1 && n % 100 !== 11) { return 0; }
     return n % 10 >= 2 && n % 10 <= 4 && (n % 100 < 10 || n % 100 >= 20) ? 1 : 2;
   },
+  lithuanian: function (n) {
+    if (n % 10 === 1 && n % 100 !== 11) { return 0; }
```

Comment:
http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html seems to suggest this should be `n % 100 < 11 && n % 100 > 19`?